package.json

```diff
@@ -32,6 +32,27 @@
     "ext": "js",
     "exec": "babel-node ./src/server/"
   },
+  "eslintConfig": {
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended"
+    ],
+    "rules": {    
+      "react/react-in-jsx-scope": "off",
```

Comment:
This is take from https://medium.com/@joelmasters/setting-up-eslint-for-nextjs-37163d4cabaa.